tf_sig_build_dockerfiles/builder.devtoolset/build_devtoolset.sh

```diff
@@ -15,7 +15,7 @@
 # ==============================================================================
 #
 # Builds a devtoolset cross-compiler targeting manylinux 2010 (glibc 2.12 /
-# libstdc++ 4.4).
+# libstdc++ 4.4) or manylinux2014 (glibc 2.17 / libstdc++ 4.8).
```

Comment:
Is this file pulled directly from tensorflow's master branch? If so, you can also make a separate PR to update it to match -- it should be kept up to date with TensorFlow's (eventually, it should only live here, but for now it should be mirrored).

Reply:
The only difference this file has with tensorflow's master branch is [e2d2d48](https://github.com/tensorflow/build/pull/52/commits/e2d2d48a1cd0a29dab87ba7900a05cb69e57715c) where I removed Python 3.6.

Reply:
I'll push commit to make it mirror the master branch's file.